requirements.txt

```diff
@@ -1,4 +1,4 @@
-jsonschema >= 4.17
+jsonschema >=4.10.0,<4.18.0
```

Comment:
If we're pinning all the way to patch, do you think we should just pick a specific version?

Reply:
Agree, changed to use specific version.